### PR TITLE
fix(curriculum): less strict calorie count step 87

### LINKED ISSUE #57147: Step 87 of calorie counter too strict
See https://forum.freecodecamp.org/t/learn-form-validation-by-building-a-calorie-counter-step-87/720435/

The test will fail if there are extra spaces at the end of certain lines, this should not cause failing

It would be great if the test could check the effect of calling `calculateCalories`, instead of regex

This is the file: https://github.com/freeCodeCamp/freeCodeCamp/blob/4524a11e57c68602fdfa27cc7ff57261e4f69a19/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e8fe3a6f022a05a04675.md

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #57147

Related to #57107 

This checks the output element for the information that is needed and also removes the `after-user-code`.

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e8fe3a6f022a05a04675.md [hunk 1]
```diff
@@ -238,7 +335,7 @@ select {
 ```js
 const calorieCounter = document.getElementById('calorie-counter');
 const budgetNumberInput = document.getElementById('budget');
-const entryDropdown = document.getElementById('entry-dropdown');
+const entryDropdownElement = document.getElementById('entry-dropdown');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e8fe3a6f022a05a04675.md:338]: why these changes to the seed?
are they reflected in previous/next steps?

Comment (a2937) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e8fe3a6f022a05a04675.md:338]: I didn't realize I changed the seed. That was a complete accident. Too many copy and replaces.

**Comment by ilenia-magoni** [comment]
Can you make the editable region smaller?
I think it could start right above `const consumedCalories`

**Comment by a2937** [comment]
Hang on. I gotta apply the changes to it's twin too.

File: curriculum/challenges/english/25-front-end-development/workshop-calorie-counter/63c9e8fe3a6f022a05a04675.md [hunk 2]
```diff
@@ -13,66 +13,207 @@ Using the same interpolation syntax, add a second `p` element with the text `con
...
+assert.strictEqual(output.children[3].innerText,"1800 Calories Consumed"); 
+
+dinnerValueElement.value = 300;
+calculateCalories(fakeEvent); 
+assert.strictEqual(output.children[3].innerText,"1400 Calories Consumed"); 
 ```
 
 Your second `p` element should have the text `${consumedCalories} Calories Consumed`.
 
+
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/workshop-calorie-counter/63c9e8fe3a6f022a05a04675.md:96]: accidental extra line

**Review by majestic-owl448 — APPROVED** [review]

**Review by Dario-DC — APPROVED** [review]
I pushed a commit to remove the console.logs. Hope you don't mind. Anyway it looks good to me :tada:  thank you!

**Comment by a2937** [comment]
I don't mind. It's really easy for me to forget to remove console.log.
